opendatafabric/src/serde/flatbuffers/proxies_generated.rs

```diff
@@ -9,8 +9,8 @@
 
 // automatically generated by the FlatBuffers compiler, do not modify
 
-use std::cmp::Ordering;
-use std::mem;
+use core::cmp::Ordering;
```

Comment:
Can you check which version of Flatbuffers compiler you are using? I we use different ones we'll be producing too much diffs.

Reply:
Mine is:
```bash
$ flatc --version
flatc version 2.0.0

$ yay -Q | grep flat
flatbuffers-git 2358-1
```
I moved to latest git version after encountering some issues in Rust codegen, but can switch back to a latest stable if it is indeed stable :)

Reply:
I'm using the latest version available on today, 2.0.6